bot/client.js

```diff
@@ -57,4 +57,17 @@ client.on('message', (message) => {
   }
 });
 
+client.on('guildMemberAdd', (member) => {
```

Comment:
Narrow scoped logic. The framework is built around running multiple controllers against each ```client.on``` while you are singularly doing this. This makes the function inflexible and a one-time use. Should this need extended further it would require a full rewrite of what you have added.